cogs/discord_datamining.py

```diff
@@ -75,17 +75,37 @@ async def commit_poster_loop(self, embed_title:str, api_url:str, repo_url:str):
             comment_text.extend([(i['commit_id'], re.sub(r"^## (.+)", r"**\1**", i['body'])) for i in data])
         self.logger.info(f"Logging info of {len(comment_text)} comments")
 
-        # Format it all into an embed
+        # Get the full description
+        descriptionunsplit =  '\n'.join([f"{i['commit']['message']} - [Link]({repo_url}commit/{i['sha']})" for i in new_commits])
+
+        # Inform the bot owner that the description is larger than allowed by discord
+        if len(descriptionunsplit)>2048:
```

Comment:
this is mainly a personal gripe, but there should be spaces:
`if len(descriptionunsplit) > 2048:`

Reply:
fixed